backend/src/test/java/gov/cdc/usds/simplereport/api/uploads/FileUploadControllerTest.java

```diff
@@ -47,7 +47,7 @@ void handleHIVResultsUpload_successful() throws IOException {
             .warnings(empty)
             .destination(Pipeline.UNIVERSAL)
             .build();
-    when(featureFlagsConfig.isHivEnabled()).thenReturn(true);
+    when(featureFlagsConfig.isHivBulkUploadEnabled()).thenReturn(true);
```

Comment:
Could we actually pull that test out and then add it back later once we figure out what the issue is? Would rather not have a test than have one that’s passing in a misleading way

Reply:
fysa @emyl3

Reply:
New issue for the test here: https://github.com/CDCgov/prime-simplereport/issues/7240